src/SampleExtension/SampleExtension.csproj

```diff
@@ -139,6 +141,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />

```

Comment:
can we just get rid of this?

Reply:
Yes we can. I can also get rid of the `logo.png` files. I didn't do it at first just to show these files get added, since it seemed a little weird.